frontend/src/app/application-forms/fields/activity-description.component.html

```diff
@@ -42,7 +42,8 @@
     </ng-container>
 
     <input id="need-temporary-improvements" type="checkbox" formControlName="needTemporaryImprovements" />
-    <label id="need-temporary-improvements-label" class="usa-input" for="need-temporary-improvements">{{ pointOfView }} would like to use temporary directional signage.</label>
+    <label id="need-temporary-improvements-label" class="usa-input" for="need-temporary-improvements">{{ pointOfView }} 
```

Comment:
This line break is unnecessary. Please revert it to maintain code style.

Reply:
@briandavidson I removed that line break yesterday